README.md

```diff
@@ -11,7 +11,7 @@ If your project doesn't use [CocoaPods](https://cocoapods.org/) yet, [follow thi
 Add the following line to your `Podfile`:
 
 ```ruby
-pod 'ACActionCable', '~> 2.0'
+pod 'ACActionCable', '~> 2.1'
```

Comment:
This could be updated to the following so it doesn't need to be updated with every minor version
```
pod 'ACActionCable', '~> 2'
```